### PR TITLE
Update GH Workflow

### PR DESCRIPTION
**Pull request by ioduok**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Test 
+
+on: 
+  pull_request: 
+    branches: 
+      - base
```
Comment (p-galligan) [inline .github/workflows/tests.yml]: This will have to be development. We want tests to run when we make a PR against dev.

Comment (ioduok) [reply .github/workflows/tests.yml]: I'm pretty sure that's why the builds kept failing. Once I corrected line 6, everything passed!

**Review by HaSistrunk — CHANGES_REQUESTED** [review]
Since we're replacing Travis with the new Github Actions workflows, you'll also want to delete the .travis.yml file and the `[![Build Status](https://travis-ci.org...` bit that references the Travis build in the README.md file.

**Review by helrond — APPROVED** [review]
I think this is okay to merge as-is, but I'm noting that there's no deploy workflow (which there wasn't in the Travis configs either). @p-galligan can you put adding that workflow on your to-do list? This should deploy to PyPI, probably when we cut a release (or whatever our general pattern is for deploying to PyPI).

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      matrix: 
+        python-version: ["3.10"] 
+        toxenv: [py310, linting] 
+
+    steps: 
+      - name: Checkout code 
+        uses: actions/checkout@v4 
+
+      - name: Set up Python ${{ matrix.python-version }} 
+        uses: actions/setup-python@v4
```
Comment (HaSistrunk) [inline .github/workflows/tests.yml]:
```suggestion
        uses: actions/setup-python@v5 
```

**Comment by p-galligan** [comment]
> I think this is okay to merge as-is, but I'm noting that there's no deploy workflow (which there wasn't in the Travis configs either). @p-galligan can you put adding that workflow on your to-do list? This should deploy to PyPI, probably when we cut a release (or whatever our general pattern is for deploying to PyPI).

Following up here as well that I a python publish workflow already exists for this repo.

**Review by p-galligan — APPROVED** [review]

**Review by HaSistrunk — APPROVED** [review]

**Pull request merged**